src/checkers/inference/solver/util/VectorUtils.java

```diff
@@ -0,0 +1,21 @@
+package checkers.inference.solver.util;
+
+import org.sat4j.core.VecInt;
+
+/**
+ * Helper class for creating VecInt and VecInt array.
+ * 
+ * @author jianchu
+ *
+ */
+public class VectorUtils {
+
+    public static VecInt asVec(int... result) {
+        return new VecInt(result);
+    }
+
+    public static VecInt[] asVecArray(int... vars) {
```

Comment:
The naming of method is misleading. The above two methods both create one clause. This method only encapsulates the clause in an array.
Usually the elements in the array are conjunction. So I tried calling `asVecArray(1,-1)` to create unsatisfiable clauses [1, -1], but actually it gives me one clause (1|-1) which is always true. 
And the class can maybe be renamed to MaxSatClauseUtil

Reply:
Add a new method to distinct below situations:
1. passing in a serials of `int`, wrap them as one `VecInt`, and then wrap this `VecInt` as a VecInt array.

2. passing in a serials of `VecInt`, wrap them as a VecInt array.